cms/server/util.py

```diff
@@ -412,8 +412,8 @@ class CommonRequestHandler(RequestHandler):
 
     def __init__(self, *args, **kwargs):
         super(CommonRequestHandler, self).__init__(*args, **kwargs)
-        self.timestamp = None
-        self.sql_session = None
+        self.timestamp = make_datetime()
+        self.sql_session = Session()
```

Comment:
So you decided to remove expire_all()? I read a bit and it looks safe to remove.

Reply:
Yes, I checked too and couldn't find a reason to keep it. Thanks for having double checked.